packages/shared_preferences/shared_preferences_web/pubspec.yaml

```diff
@@ -0,0 +1,28 @@
+name: shared_preferences_web
+description: Web platform implementation of shared_preferences
+author: Flutter Team <flutter-dev@googlegroups.com>
+homepage: https://github.com/flutter/plugins/tree/master/packages/shared_preferences/shared_preferences_web
+version: 0.0.1
```

Comment:
Nit: prefer the initial release to be `0.1.0` to get a better score on pub

Reply:
Done.